_data/events/London-TAB.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Global Day of Code Retreat @TAB",
+  "url": "https://www.eventbrite.co.uk/e/global-day-of-code-retreat-tab-london-tickets-39248331817",
+  "moderators": [
+    "@TheAppBusiness"
```

Comment:
Please provide the actual moderators as well and not just the sponsoring company. As of #159, we will remove companies from the `moderators` field and move them to their own field and we want to avoid having an empty moderators field :)

Reply:
Will do.